views/layouts/email_l10n_mockups.hbs

```diff
@@ -2,8 +2,8 @@
 <html lang="{{ locale }}">
   <head>
     <meta charset="utf-8" />
-    <link rel="stylesheet" href="/dist/app.min.css">
-    <link rel="stylesheet" href="/dist/email_l10n_testing.min.css">
+    <link rel="stylesheet" href="/css/legacy/app.css">
+    <link rel="stylesheet" href="/css/legacy/email_l10n_testing.css">
```

Comment:
intentional? or do we want to keep this pointing to `/dist/` ?

Reply:
intentional because the /email-l10n pages rely on it. Perhaps we could condense these into one file and strip out the excess gunk though.